src/vs/base/browser/htmlContentRenderer.ts

```diff
@@ -90,6 +90,29 @@ function _renderHtml(content: IHTMLContentElement, options: RenderOptions = {}):
 		const withInnerHTML = new TPromise(c => signalInnerHTML = c);
 
 		const renderer = new marked.Renderer();
+		renderer.image = (href: string, title: string, text: string) => {
+			let attributes = '';
+			if (title) {
+				const splitted = title.split('|');
+				title = splitted[0];
```

Comment:
Looks like the actual title is getting lost now. Add it back into the attributes